app/javascript/components/host-initiator-form/index.jsx

```diff
@@ -32,14 +32,21 @@ const HostInitiatorForm = ({ redirect, storageManagerId }) => {
   };
 
   const onCancel = () => {
-    const message = __('defining of host initiator was cancelled by the user');
+    const message = __('Defining of host initiator was cancelled by the user.');
```

Comment:
Minor but I think `Creation of host initiator` makes more sense

Reply:
Sure.